yarn.lock

```diff
@@ -1113,7 +1081,7 @@
 
 "@aws-sdk/client-location@3.186.1":
   version "3.186.1"
-  resolved "https://registry.npmjs.org/@aws-sdk/client-location/-/client-location-3.186.1.tgz#e3be4913f5673b8afde2f11c5c8784d5425e71b7"
+  resolved "https://registry.yarnpkg.com/@aws-sdk/client-location/-/client-location-3.186.1.tgz#e3be4913f5673b8afde2f11c5c8784d5425e71b7"
```

Comment:
Anyone know how to force my yarn to stop using yarnpkg.com? I've tried a bunch of things and it keeps ignoring me

Reply:
Have you tried `yarn config set registry https://registry.npmjs.org/` and running the install again?